transaction_csv_cleanup.py

```diff
@@ -67,7 +67,7 @@ def clean_data(file):
     has_headers = g_config["has_headers"]
     output_data = []
     with open(file) as transaction_file:
-        transaction_reader = csv.reader(transaction_file, delimiter = delim)
+        transaction_reader = csv.reader(transaction_file, delimiter=str(delim).encode('utf-8'), quotechar=str(u'"').encode('utf-8'))
```

Comment:
Messy. I think I'd rather encode things when getting them from the config file. What's the quotechar section?

Reply:
Also...I'm not getting this error? From a quick Google it seems to be a Python 2.x issue.

Reply:
Py2.x ... this is so odd! I must try to uninstall/purge python from my system and make an explicit py3 installation. Let's see how much of this goes away then. I'll respond with results! Let's keep this unmerged until then. 

I agree that it's probably messy. I was trying to make it work using minimal changes; a more elegant solution would probably belong somewhere else in the code. My system was complaining that the delimiter character wasn't in Unicode, so this is what I found as a solution to convert it to U. The quotechar was part of what I found; it's also a parameter like the delimiter and it reminded me that things like Excel also allow me to specify something other than the typical double quote as the quotation mark, for instance just a single quote instead. If we keep it, it should be treated elegantly like the unicode-ification of the delimiter character, and also included in the conf file.

Reply:
I think it's straightforward to import as unicode. Think configparser has an encode function that can do this.

Reply:
Fixed this one by modifying line 26 to 
`    config.read(conf_files, encoding = "utf-8")
`
Will commit to mine later. Don't think I can modify a pull request. Or maybe I can...

Reply:
I can! I cut out the quotechar bit for now, but will look at it again when the base stuff is actually working cross platform!